client/src/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -29,6 +32,29 @@ interface Props {
 
 export default function Login({ handleSubmit }: Props): JSX.Element {
   const classes = useStyles();
+  const { updateLoginContext } = useAuth();
+  const { updateSnackBarMessage } = useSnackBar();
+
+  const handleClick = () => {
+    login('lukmanbioku@gmail.com', '1234567')
+      .then((data) => {
+        if (data.error) {
+          // setSubmitting(false);
+          updateSnackBarMessage(data.error.message);
+        } else if (data.success) {
+          updateLoginContext(data.success);
+        } else {
+          // should not get here from backend but this catch is for an unknown issue
+          console.error({ data });
+
+          // setSubmitting(false);
+          updateSnackBarMessage('An unexpected error occurred. Please try again');
+        }
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  };
```

Comment:
`DemoLogin` button looks good! 👍🏼 However, we can be more [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) and refactor this into a component of its own (the way it is currently usually leads to more work - for example, you would have to apply the suggestion above in two different places). You can receive any `Formik` related infromation through props or via the [`useFormikContext` helper](https://formik.org/docs/api/useFormikContext)! 🎉